1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -140,7 +140,7 @@ function generate_kubernetes_config() {
       <jenkinsUrl>http://${JENKINS_SERVICE_HOST}:${JENKINS_SERVICE_PORT}</jenkinsUrl>
       <jenkinsTunnel>${JNLP_HOST}:${JNLP_PORT}</jenkinsTunnel>
       <credentialsId>1a12dfa4-7fc5-47a7-aa17-cc56572a41c7</credentialsId>
-      <containerCap>10</containerCap>
+      <containerCap>100</containerCap>
```

Comment:
don't bother for the v1 version, we are no longer publishing those images

Reply:
okay, i did it just because it was there... i'll revert the change. thanks!

Reply:
reasonable assumption on your part for sure :-)

just another snowflake of ours in this space